### PR TITLE
Use input artist_id, not returned artist_id for the album artist check

### PR DESCRIPTION
**Pull request by darrell-k**
See https://forums.slimdevices.com/forum/developer-forums/developers/1733221-bugs-in-album-query-in-r1730788382?p=1733452#post1733452

### CODE REVIEW

File: Slim/Menu/BrowseLibrary/Releases.pm [hunk 1]
```diff
@@ -42,6 +42,7 @@ sub _releases {
 		$artistIds[0] =~ /artist_id:(\d+)/;
 		$artistId = $1;
 	}
+$log->error("DK \$artistId=" . Data::Dump::dump($artistId));
```
Comment (michaelherger) [inline Slim/Menu/BrowseLibrary/Releases.pm]: Ahm... 😀

File: Slim/Menu/BrowseLibrary/Releases.pm [hunk 2]
```diff
@@ -125,10 +126,11 @@ sub _releases {
 			my $albumArtist = Slim::Schema->first('ContributorAlbum', {
 				album => $_->{id},
 				role  => Slim::Schema::Contributor->typeToRole('ALBUMARTIST'),
-				contributor => { '!=' => $_->{artist_id} }
+				contributor => { '!=' => $artistId }
 			});
 
 			if (!$albumArtist) {
+$log->error("DK not albumArtist");
```
Comment (michaelherger) [inline Slim/Menu/BrowseLibrary/Releases.pm]: Oh...

**Review by michaelherger — CHANGES_REQUESTED** [review]
I shouldn't do this on the phone. But those two lines aren't wanted, are they?

**Comment by darrell-k** [comment]
Well I like them, they tell me what's going on 😀!

**Review by michaelherger — APPROVED** [review]
Let's give that a try. Thanks!

**Pull request merged**